app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -346,7 +348,8 @@ class AdyenPaymentFragment : DaggerFragment(), AdyenPaymentView {
   override fun getPaymentDetails(): Observable<RedirectComponentModel> = paymentDetailsSubject!!
 
   override fun getSupportClicks(): Observable<Any> {
-    return Observable.merge(RxView.clicks(layout_support_logo), RxView.clicks(layout_support_icn))
+    return Observable.merge(RxView.clicks(layout_support_logo), RxView.clicks(layout_support_icn),
+        RxView.clicks(layout_support_icn_top_up), RxView.clicks(layout_support_logo_top_up))
```

Comment:
Why have topup here?
Plus, not yours but there should be a different method for each view and the presenter is the one that should do the merge